src/viewer.js

```diff
@@ -107,12 +111,18 @@ class Viewer {
   createInitParams_() {
     const parsedViewerUrl = parseUrl(window.location.href);
 
-    // TODO (chenshay): set more init params.
     const initParams = {
       origin: parsedViewerUrl.origin,
     };
 
     if (this.referrer_) initParams.referrer = this.referrer_;
+    if (this.prerender_) {
+      initParams.visibilityState = 'prerender';
+      initParams.prerenderSize = 1;
+    }
+    if (this.prerender_ != undefined && !this.prerender_) {
+      initParams.prerenderSize = 0;
+    }
```

Comment:
Is it necessary to set prerenderSize to 0? If you're not prerendering does it matter what value it has?